tests/test_sessions.py

```diff
@@ -9,6 +13,40 @@
 from . import SpanRecorder
 
 
+@contextmanager
+def environment(**overrides: Optional[str]) -> Generator[None, None, None]:
```

Comment:
I should really send this function as a PR to the standard library so I don't have to keep writing and testing it in every project ;)